modules/identity/src/Volo.Abp.PermissionManagement.Domain.Identity/Volo/Abp/PermissionManagement/Identity/UserDeletedEventHandler.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+using Volo.Abp.Authorization.Permissions;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Entities.Events.Distributed;
+using Volo.Abp.EventBus.Distributed;
+using Volo.Abp.Users;
+
+namespace Volo.Abp.PermissionManagement.Identity;
+
+public class UserDeletedEventHandler :
+    IDistributedEventHandler<EntityDeletedEto<UserEto>>,
+    ITransientDependency
+{
+    protected IPermissionManager PermissionManager { get; }
+
+    public UserDeletedEventHandler(IPermissionManager permissionManager)
+    {
+        PermissionManager = permissionManager;
+    }
+
+    public async Task HandleEventAsync(EntityDeletedEto<UserEto> eventData)
```

Comment:
Since `PermissionManager` is a domain service (that doesn't manage UOW), it is better to make that method UOW (and virtual).